oauth/localhost.pem

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE-----
+MIIEazCCAtOgAwIBAgIRAKFrpccs/qdwuUnba+w8hlMwDQYJKoZIhvcNAQELBQAw
+gZ0xHjAcBgNVBAoTFW1rY2VydCBkZXZlbG9wbWVudCBDQTE5MDcGA1UECwwwYW1h
+bmRhaEBBbWFuZGFzLU1hY0Jvb2stUHJvLmxvY2FsIChBbWFuZGEgSHVhbmcpMUAw
+PgYDVQQDDDdta2NlcnQgYW1hbmRhaEBBbWFuZGFzLU1hY0Jvb2stUHJvLmxvY2Fs
+IChBbWFuZGEgSHVhbmcpMB4XDTIxMDkyMjIyMDgzMFoXDTIzMTIyMjIzMDgzMFow
+ZDEnMCUGA1UEChMebWtjZXJ0IGRldmVsb3BtZW50IGNlcnRpZmljYXRlMTkwNwYD
+VQQLDDBhbWFuZGFoQEFtYW5kYXMtTWFjQm9vay1Qcm8ubG9jYWwgKEFtYW5kYSBI
+dWFuZykwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDHDT/NcLhkTMn6
+M7gIcZEaQXDNZvYnUfS9UkvtdMpBDz2pbbYbQ1xB6RWJXi/0356SW3VMiF+unWeG
+iQd+QG3Mgu/0a9+s+AzFP3iyxp++1+5vxR7sKuy9at0jvvnhcUG09JXuVt9xv8ur
+HBQVkDDS+7WhPG2Ku6z6MRlPtqVsHgOgoui6vzCYfhkR2Ao3UwWDQEi51xZzE0Sr
+ZdedxRFscCJSpuT/+5lR+hrKMCUdXOh83qPD2aeC6BNBRIEoYJokAnMF0BQrd/jU
+S7Ez7l//zwfkKKLamoUhFbI627Phv/gE1+bmo92gp23FCHyDlFCSRV3R4iyLLLq8
+0YGzfy6hAgMBAAGjXjBcMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEF
+BQcDATAfBgNVHSMEGDAWgBRMckCaGvFvcXfB+9CooOfjPbB3JjAUBgNVHREEDTAL
+gglsb2NhbGhvc3QwDQYJKoZIhvcNAQELBQADggGBAMBYqmNrCBMoXqMQL1HRfXXl
+jWCH7JSsAbYAhiaMCxjErgwFxbT//YwwlDqsIQ+KJ7lSBBE2J5MDbnxKasXWttrC
+9xTEbCU6XezPRXAMYbd7KCHi1mP+O7Wuevk8INrHpuK+cPOOPLH4B23uU5FAIhC9
+qBnLvVga5ZogW2lNfTCJvh8YHJrf2jGQH2tht6uAtHQbYQXrpZb1FxVWCu6U0Njv
+j6sGnZuTbmrGGCkeIT+RPEjNs7cKJ+rfe9jYkE/KHg+S3wYWm4sM6dQFBeyVgb2E
+23Z5YmfjdQu8yvLV/3ty6Igvv19xb12w+XfHOs+fN8vet6dxhOcdr3QsLu0RltED
+o5wNlJzNp7n3S8QdhvB5240SB0v3eQ75SPfhEWmO8S1LaHw4/0qFMFXTMxlPatMe
+REDHuCenFbKikQfK4v9Y/1nFOnqe3o5l+Z6wE2Ey84AOJua03VJX2oi3vpMlyonI
+BJTbyw6gVQdRI+Q3/h363u/eCLN6j409gi8Ll1jFLg==
+-----END CERTIFICATE-----
```

Comment:
I think Cert and Key shouldn't be part of the repo and instead be generated on the user's system.